02.calendar/cal.rb

```diff
@@ -0,0 +1,57 @@
+#!/Users/heureux/.rbenv/versions/2.6.5/bin/ruby
+
+require 'optparse'
+require 'date'
+
+target_year = ''
+target_month = ''
+current_year = Date.today.year
+current_month = Date.today.month
+
+days = ['日','月','火','水','木','金','土']
+last_date = ''
+cal_title = ''
+day_header = ''
+
+# オプションの値を取得
+opt = OptionParser.new
+opt.on('-y')
+opt.on('-m')
+opt.parse!(ARGV)
+
+if ARGV.empty?
+  argument = [current_year.to_s,current_month.to_s]
+else
+  argument = ARGV
+end
+
+target_year = argument[0]
+target_month = argument[1]
+
+last_date = Date.new(target_year.to_i, target_month.to_i, -1).strftime('%d')
+
+# カレンダータイトル
+cal_title.concat('      ' + target_month + '月 ' + target_year)
+
+# 曜日の表示
+days.each do |day|
+  day_header.concat(day + ' ')
+end
+
+puts cal_title
+puts day_header
+
+# 日付部分の出力
+# 初日の曜日の数字を確認し、数字x3スペース開ける
+first_day_date = Date.new(target_year.to_i,target_month.to_i,1).wday
+print ' ' * first_day_date * 3
+
+(1..last_date.to_i).each do |date|
+  date < 10 ? pre_space = ' ' : pre_space = ''
```

Comment:
```suggestion
  pre_space = date < 10 ? ' ' : ''
```

みたいな書き方の方が主流ですね〜。

一方で、Rubyにはこんなメソッドがあるので、これを利用するのも一手です。

https://docs.ruby-lang.org/ja/latest/class/String.html#I_RJUST

Reply:
rjustを使って書き換えました。